<launch>

  <node name="image1" pkg="image_view" type="image_view">
    <remap from="image" to="/camera1/tracked_image" />
  </node >

  <node name="image2" pkg="image_view" type="image_view">
    <remap from="image" to="/camera2/tracked_image" />
  </node >

  <node name="image3" pkg="image_view" type="image_view">
    <remap from="image" to="/camera3/tracked_image" />
  </node >

  <node name="image4" pkg="image_view" type="image_view">
    <remap from="image" to="/camera4/tracked_image" />
  </node >

</launch>